src/Symfony/Component/Cache/Serializer/NullSerializer.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Cache\Serializer;
+
+use Symfony\Component\Cache\SerializerInterface;
+
+class NullSerializer implements SerializerInterface
```

Comment:
Maybe this should be called `IdentitySerializer`, I expected a `NullSerializer` to always return `null`.